apps/src/p5lab/P5Lab.js

```diff
@@ -359,6 +359,10 @@ P5Lab.prototype.init = function(config) {
 
     this.studioApp_.init(config);
 
+    if (startInAnimationTab(getStore().getState())) {
```

Comment:
i'm not sure i understand how moving this conditional from line 449 to line 362 fixed the issue we were seeing since `this.studioApp_.init(config)` is on line 360

Reply:
it's a little hard to follow because P5Lab.init() has *a lot* going on, but `this.studioApp_.init(config)` is inside another function (`onMount`), which is defined on line 324. So I'm moving the conditional to be after `this.studioApp_.init(config)` in that function, which is passed to P5LabView and called after that component is mounted